lib/gtl_formatter.rb

```diff
@@ -132,6 +169,18 @@ def self.format_icon_column(view, row, controller, clickable)
      :icon2 => icon2}.compact
   end
 
+  def self.os_info_details(item)
+    return nil unless item
+
+    "svg/os-#{item.os_image_name}.svg"
```

Comment:
could `item.os_image_name` be nil?   Actually, it looks like the [backend](https://github.com/ManageIQ/manageiq/blob/3b4f5050af8a90515f066927d63d239da69be481/app/models/vm_or_template.rb#L558-L565) code will most likely return [unknown](https://github.com/ManageIQ/manageiq/blob/3b4f5050af8a90515f066927d63d239da69be481/app/models/operating_system.rb#L76-L126) in those situations.

Reply:
it can't be nil - at worst it will return "unknown"

Reply:
yeah, that's what i said in the second sentence... let me strikethrough the first one.